### ISSUE TITLE
Is there any option to set viewport equal to the boundary?

### ISSUE THREAD

**Issue by shreeramk** [Foliotek/Croppie#382]
@thedustinsmith : One way to do is below. But, I was expecting a default option for viewport to be equal to the boundary.

`<div class="upload"></div>`

```
$('.upload').croppie({
	url: '/tmp/img.png',
	showZoomer: false,
	viewport: {
		width: $('.upload').width(),
		height: $('.upload').height()
	}
});
```

**Comment by thedustinsmith** [comment]
No, there's no built in way to do this.  The defaults were what worked when we first set this up, as of this point in the project they seem really arbitrary.  As of right now, I don't see much reason in changing them though.  It's simple enough to just tweak the viewport/boundary options.
